specification/under_development/profiles/Base_Profile.md

```diff
@@ -0,0 +1,522 @@
+# Base Profile
+
+This profile defines a subset of the QIR specification to support a coherent set
+of functionalities and capabilities that might be offered by a quantum backend.
+Like all profile specifications, this document is primarily intended for
+[compiler backend](https://en.wikipedia.org/wiki/Compiler#Back_end) authors as
+well as contributors to the [targeting
+stage](../Compilation_And_Targeting.md#targeting) of the QIR compiler.
+
+The Base Profile specifies the minimal requirements for executing a quantum
+program. Specifically, to execute a Base Profile compliant program, a backend
+needs to support the following:
+
+1. It can execute a sequence of quantum instructions that transform the quantum
+   state.
+2. It supports measuring the state of each qubit at the end of the program.
+3. It produces one of the specified [output schemas](../output_schemas/).
+
+These functionalities are necessary and sufficient for computations that
+fundamentally consist of unitary transformations of the quantum state as well as
+measurements at the end of the program. More details about each of the bullets
+are outlined below.
+
+**Bullet 1: Quantum transformations** <br/>
+
+The set of available instructions that transform the quantum state may vary
+depending on the targeted backend. The profile specification defines how to
+leverage and combine the available instructions to express a program, but does
+not dictate which quantum instructions may be used. Targeting a program to a
+specific backend requires choosing a suitable profile and quantum instruction
+set (QIS). Both can be chosen largely independently, though certain instruction
+sets may be incompatible with this (or other) profile(s). The section on the
+[quantum instruction set](#quantum-instruction-set) defines the requirements for
+a QIS to be compatible with the Base Profile. More information about the role of
+the QIS, recommendations for front- and backend providers, as well as the
+distinction between runtime functions and quantum instructions can be found in
+[this document](../Instruction_Set.md).
+
+**Bullet 2: Measurements** <br/>
+
+The second requirement should be taken to mean that a Base Profile compliant
+program does *not* apply instructions to a qubit after it has been measured;
+instructions that result in a unitary transformation of the quantum state must
+be applied before performing any irreversible actions such as measurements. It
+specifically also implies the following:
+
+- There is no need for the quantum processor ([QPU](../Execution.md)) to be able
+to measure only a subset of all available qubits at a time.
+- Executing a Base Profile compliant program does not require support for
+applying quantum instructions dependent on measurement outcomes.
+
+**Bullet 3: Program output** <br />
+
+The specification of QIR and all its profiles needs to permit to accurately
+reflect the program intent. This includes being able to define and customize the
```

Comment:
FWIW, I don't follow the intent of this first sentence.

Reply:
Perhaps something like this would be more clear:

The QIR specification and its profiles describe a mechanism to accurately reflect program intent with regard to program output. The Base Profile specification requires explicitly defining program output by expressing which values/measurements are returned by the program and in which order. How to express this is defined in the section on [output recording](https://github.com/qir-alliance/qir-spec/blob/base-profile/specification/under_development/profiles/Base_Profile.md#output-recording).

Reply:
I took @rtvuser1's rephrasing.